Documentation/StyleGuide.md

```diff
@@ -110,8 +135,38 @@ reader are not expected to know each other personally, so avoid overly familiar
 terms or colloquialisms such as "let's."
 
 When writing specific instructions that a reader must follow exactly in order to
-accomplish a task, use the second person ("you", "your", etc.) Otherwise, avoid
-using the second person.
+accomplish a task, use the second person ("you", "your", etc.).
+
+#### Articles
+
+When writing articles or curating content, keep the structure simple and
+relatively flat. Place articles in topic groups alongside the symbols.
+
+Some basic rules to follow when creating an article are:
+- Begin an article title with a _gerund_ (a verb ending in "ing").
+- After the title, include a single sentence that begins with a verb and quickly
+describes what the article covers.
+- Include an overview to serve as an introduction to the article. If required,
+include any setup or configuration tasks as a part of your overview.
+- Start section headings with an imperative verb.
+- Always follow a section heading with some text to setup the code or problem
+you want to solve.
+- Match the names of articles and the files that contain them using kebab-case.
+
+#### API collections
+
+To organize related subsets of symbols, articles, and other content, use an API
+collection.
+
+Some basic rules to follow when creating an API collection are:
+- Begin the collection title with a noun that describes what the items in the
+collection have in common.
+- After the title, include a single sentence that describes the items in the
+collection.
+- Optionally, include an overview to the collection.
+- Organize the symbols under topic group headings. Begin a topic group heading
+with a gerund.
+- Match the names of collections and the files that contain them using kebab-case.
```

Comment:
```suggestion
- The filename for your API collection should match the title of the collection page, except in lowercase characters, ignoring punctuation, and replacing spaces with hyphens (kebab-case). For example, if the title of the collection page is _Event tags_, the filename would be `event-tags.md`.
```

Reply:
Should there be a separate section on file-naming?

Reply:
To consolidate the instructions for articles and collections? Maybe. We'd need to state how to spell filenames for other kinds of documentation _and_ for source files.